docs/ref/core.rst

```diff
@@ -402,3 +402,15 @@ Information about the locale and message are available from the event.
    i18n.on('missing', (event) => {
       alert(`alert(`Translation in ${event.locale} for ${event.id} is missing!`)`)
    })
+
+getting-translation
+-------
+
+Triggered when a translation is requested with ``i18n._``, regardless the message exists or not in the active locale's messages.
```

Comment:
@hajaj-medigate This got me thinking: it doesn't make sense to get a translation for something that is missing. Wouldn't it be better if the event was emitted only if it exists in the active locale's meesages?
Meaning the `this.emit(...)` call in `packages/core/src/i18n.ts` would be surrounded by an `else` statement.

Reply:
No, because `getting-translation` is called when a translation is requested, it doesn't matter if it missing or not.

@Martin005 WDYT?

Reply:
@hajaj-medigate But if the translation is missing there is no point of "getting" it, right? So from my point of view, it makes sense if the `getting-translation` was called only when the translation is not missing.

Reply:
My use case is more like `requesting-translation` rather than `getting-translation`
should I rename it?

@Martin005

Reply:
@Martin005 @andrii-bodnar  - any thoughts?

Reply:
@hajaj-medigate actually, I have some doubts regarding adding a new event and how it can be useful for other Lingui users